nts-pool-monitor/src/cli.rs

```diff
@@ -0,0 +1,190 @@
+use super::daemon_tracing::LogLevel;
+use std::path::PathBuf;
+use std::str::FromStr;
+
+const USAGE_MSG: &str = "\
```

Comment:
These docs should all be updated (or removed until they become stable as an alternative I guess)